specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/integrations.json

```diff
@@ -0,0 +1,653 @@
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page"
+        }
+      }
+    },
+    "Integration": {
+      "type": "object",
+      "description": "Integration over a given scope",
+      "properties": {
+        "properties": {
+          "description": "Properties of an integration",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IntegrationProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "IntegrationProperties": {
+      "type": "object",
+      "description": "Describes properties of an integration",
+      "properties": {
+        "tenantId": {
+          "description": "The tenantId (GUID)",
+          "readOnly": true,
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Display name of the integration",
+          "type": "string"
+        },
+        "isDisabled": {
+          "description": "Defines whether the integration is active/inactive",
+          "type": "string"
+        },
+        "integrationType": {
+          "type": "string",
+          "description": "The integration platform type, defines the source of the integration e.g. ServiceNow",
+          "enum": [
+            "ServiceNow"
+          ],
+          "x-ms-enum": {
+            "name": "IntegrationType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "ServiceNow",
+                "description": "The source of the integration type definition is ServiceNow"
+              }
+            ]
+          }
+        },
+        "integrationRootDetails": {
+          "$ref": "#/definitions/IntegrationRootDetails"
+        },
+        "excludedScopes": {
+          "type": "array",
+          "description": "Excluded scopes, filter out the descendants of the scope (on management scopes)",
+          "items": {
+            "description": "The excluded scope",
+            "type": "string"
+          },
+          "x-ms-identifiers": []
+        },
+        "includeMemberScopes": {
+          "type": "string",
+          "description": "Defines whether the integration is management scope integration (master connector as a single scope or management scope)"
+        },
+        "metadata": {
+          "$ref": "#/definitions/IntegrationMetadata"
+        },
+        "integrationAdditionalData": {
+          "type": "object",
+          "description": "Additional data",
+          "$ref": "#/definitions/IntegrationAdditionalData"
+        }
+      },
+      "required": [
+        "displayName",
+        "integrationType"
+      ]
+    },
+    "IntegrationRootDetails": {
+      "type": "object",
+      "description": "The integration root details",
+      "properties": {
+        "resourceId": {
```

Comment:
>resourceId [](http://example.com/codeflow?start=9&length=10)

Use this: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details), if it is an ARM resource id.

Reply:
?